src/main/java/com/direwolf20/buildinggadgets/common/entities/BlockBuildEntity.java

```diff
@@ -21,14 +21,17 @@
 import java.util.Optional;
 
 public class BlockBuildEntity extends EntityBase {
-    private static final DataParameter<Integer> toolMode = EntityDataManager.<Integer>createKey(BlockBuildEntity.class, DataSerializers.VARINT);
+
+    public static final int MODE_PLACE = 1;
+    public static final int MODE_REMOVE = 2;
+    public static final int MODE_REPLACE = 3;
+
+    private static final DataParameter<Integer> TOOL_MODE = EntityDataManager.createKey(BlockBuildEntity.class, DataSerializers.VARINT);
```

Comment:
I'm Pretty sure there are already enum data parameters...

Reply:
Can you clarify that? I just renamed this constant, from `toolMode` to `TOOL_MODE`.